src/Pickles/Pickles.Test/DocumentationBuilders/HTML/WhenFormattingStep.cs

```diff
@@ -181,5 +181,123 @@ public void Tables_are_formatted_as_list_items_with_tables_internal()
 
             Check.That(expected).IsDeeplyEqualTo(actual);
         }
+
+        [Test]
+        public void Comments_are_displayed_above_their_related_step()
+        {
+            var step = new Step
+            {
+                Keyword = Keyword.Given,
+                NativeKeyword = "Given ",
+                Name = "a simple step",
+                TableArgument = null,
+                DocStringArgument = null,
+                Comments = new List<Comment>()
+                {
+                    new Comment()
+                    {
+                        Text = "    # A simple comment",
+                        Type = CommentType.StepComment
+                    }
+                }
+            };
+
+            var formatter = Container.Resolve<HtmlStepFormatter>();
+            XElement actual = formatter.Format(step);
+
+            XNamespace xmlns = XNamespace.Get("http://www.w3.org/1999/xhtml");
```

Comment:
Do me a favor and introduce a private field with the namespace and use it in all tests in this class. Thanks!